dags/multipod/configs/maxtext_gce_config.py

```diff
@@ -115,7 +115,7 @@ def get_maxtext_end_to_end_test_config(
 
   test_platform = common.Platform.GCE
   set_up_cmds = common.setup_maxtext(test_mode, test_platform)
-  run_model_cmds = (f"cd /tmp/maxtext && bash end_to_end/{test_script}.sh",)
+  run_model_cmds = (f"cd /tmp/maxtext && bash end_to_end/tpu/{test_script}.sh",)
```

Comment:
Not sure if this will always be tpu but for now I guess is fine

Reply:
This is for TPU only, `job_test_config` below uses TPU.